### PR TITLE
TM-3732 update/eas 

### PR DESCRIPTION
**Pull request by andrews-john**
added functionality for location data to be added to emp agenda search cards and rows

Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/742)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/331)

### CODE REVIEW

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx [hunk 1]
```diff
@@ -28,6 +30,8 @@ const EmployeeAgendaSearchCard = ({ isCDO, result, showCreate, viewType, showEdi
 
   // handles error where some employees have no Profile
   const employeeHasCDO = !isNil(get(person, 'cdo'));
+  // const formatLoc = getCustomLocation(currentLoc, currentPos);
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:33]: did this not work?

**Comment by elizabeth-jimenez** [comment]
Adding WIP label for cleaning this PR up a bit - remove when done cleaning

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx (hunk 1 above)
Comment (andrews-john) [reply src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:33]: looks like that was a part i commented out that i dont need anymore. I condensed this by doing const formatLoc = getCustomLocation(get(currentAssignment, 'location') || FALLBACK, currentPos);. Just less lines, figured it was a way I could speed up the review process. Ill keep looking for comments i missed

File: src/sass/_employeeSearch.scss [hunk 2]
```diff
@@ -237,6 +237,14 @@
         .org-fa-arrow {
           padding: 0 9px;
         }
+        .location-label-card{
+          // width: 150px;
+          width: 37%;
+        }
+        .location-data-card{
+          // width: 150px;
```
Comment (SidelineCory24) [inline src/sass/_employeeSearch.scss:245]: if you're not still testing out these widths, you can remove the comments

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx [hunk 3]
```diff
@@ -4,17 +4,21 @@ import FA from 'react-fontawesome';
 import { Handshake } from 'Components/Ribbon';
 import LinkButton from 'Components/LinkButton';
 import { get, isNil } from 'lodash';
-import { formatDate } from 'utilities';
+import { formatDate, getCustomLocation } from 'utilities';
 import { FALLBACK } from '../EmployeeAgendaSearchCard/EmployeeAgendaSearchCard';
 
+
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:10]: can remove

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx [hunk 4]
```diff
@@ -4,17 +4,21 @@ import FA from 'react-fontawesome';
...
 const EmployeeAgendaSearchRow = ({ isCDO, result, showCreate, viewType, showEdit }) => {
   // will need to update during integration
+  // add  back to below
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:13]: what exactly needs to be added back to the below? If it's already been added you can remove this comment

**Comment by SidelineCory24** [comment]
need to resolve conflicts

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx [hunk 5]
```diff
@@ -6,18 +6,19 @@ import { Handshake } from 'Components/Ribbon';
...
 const EmployeeAgendaSearchCard = ({ isCDO, result, showCreate, viewType, showEdit }) => {
   // will need to update during integration
-  const { person, currentAssignment, hsAssignment, agenda } = result;
+  const { person, currentAssignment, agenda, hsAssignment } = result;
   const agendaStatus = get(agenda, 'status') || FALLBACK;
   // const author = get(result, 'author') || 'Coming soon';
   const bidder = get(person, 'fullName') || FALLBACK;
   const cdo = get(person, 'cdo.name') || FALLBACK;
-  const currentPost = get(currentAssignment, 'orgDescription') || FALLBACK;
+  const currentPos = get(currentAssignment, 'orgDescription') || FALLBACK;
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:20]: can change this back to currentPost

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx [hunk 6]
```diff
@@ -28,6 +29,7 @@ const EmployeeAgendaSearchCard = ({ isCDO, result, showCreate, viewType, showEdi
 
   // handles error where some employees have no Profile
   const employeeHasCDO = !isNil(get(person, 'cdo'));
+  const currentPost = `${formatLoc} (${currentPos})`;
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:32]: can make this `currentPost$`

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx [hunk 7]
```diff
@@ -4,17 +4,19 @@ import FA from 'react-fontawesome';
...
 const EmployeeAgendaSearchRow = ({ isCDO, result, showCreate, viewType, showEdit }) => {
   // will need to update during integration
   const { person, currentAssignment, hsAssignment, agenda } = result;
   const agendaStatus = get(agenda, 'status') || FALLBACK;
   // const author = get(result, 'author') || 'Coming soon';
+
   const bidder = get(person, 'fullName') || FALLBACK;
   const cdo = get(person, 'cdo.name') || FALLBACK;
-  const currentPost = get(currentAssignment, 'orgDescription') || FALLBACK;
+  const currentPos = get(currentAssignment, 'orgDescription') || FALLBACK;
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:18]: can change this back to currentPost and update where needed

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx [hunk 8]
```diff
@@ -26,6 +28,7 @@ const EmployeeAgendaSearchRow = ({ isCDO, result, showCreate, viewType, showEdit
 
   // handles error where some employees have no Profile
   const employeeHasCDO = !isNil(get(person, 'cdo'));
+  const currentPost = `${formatLoc} (${currentPos})`;
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:31]: can make this currentPost$ and update where needed

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx [hunk 9]
```diff
@@ -16,9 +16,11 @@ const EmployeeAgendaSearchRow = ({ isCDO, result, showCreate, viewType }) => {
   const { person, currentAssignment, hsAssignment, agenda } = result;
   const agendaStatus = get(agenda, 'status') || FALLBACK;
   // const author = get(result, 'author') || 'Coming soon';
+
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:19]: can remove🧼

**Comment by SidelineCory24** [comment]
need to verify correct data is coming from WS, see [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/331)

**Comment by SidelineCory24** [comment]
Will approve this PR, however the [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/742) and [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/331) PRs are blocked by as WS issue.

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

**Pull request merged**